mnemonic-core/src/test/java/org/apache/mnemonic/DurablePersonNGTest.java

```diff
@@ -78,25 +79,72 @@ public boolean reclaim(Long mres, Long sz) {
         }
 
         person = PersonFactory.create(act);
+        int size = rand.nextInt(1024 * 1024) + 1024 * 1024;
+/////////////////////////if size is smaller than remaining act capacity////
+
+        mbh = act.createBuffer(size);
+        if (mbh != null) {
+          Assert.assertNotNull(mbh);
+          for (int i = 0; i < size; i++) {
+            mbh.get().put((byte) rand.nextInt(255));
+          }
+          person.setPicture(mbh, true);
+          mbh.destroy();
+        }
+        mch = act.createChunk(size);
+        if (mch != null) {
+          Assert.assertNotNull(mch);
+         /* for (int i = 0; i < size; i++) {
+            mch.get().put((byte) rand.nextInt(255));
+          }*/
+          person.setFingerprint(mch, true);
+          mch.destroy();
```

Comment:
Please do not destroy it since the destroy means free the underlying memory resources hold by mch.